geode-protobuf/src/main/java/org/apache/geode/protocol/protobuf/utilities/ProtobufResponseUtilities.java

```diff
@@ -50,47 +38,14 @@
    * @param ex - exception which should be logged
    * @return An error response containing the first three parameters.
    */
-  public static ClientProtocol.Response createAndLogErrorResponse(String errorMessage,
+  public static ClientProtocol.ErrorResponse createAndLogFailureResponse(String errorMessage,
```

Comment:
I think those were to changes that were made independently. Syncing this up right now